### PR TITLE
EVG-14751 Update duration calculation for cedar test results

### PR DESCRIPTION
**Pull request by khelif96**

### CODE REVIEW

File: rest/model/test.go [hunk 1]
```diff
@@ -101,7 +101,8 @@ func (at *APITest) BuildFromService(st interface{}) error {
 		at.StartTime = utility.ToTimePtr(v.Start)
 		at.EndTime = utility.ToTimePtr(v.End)
 		duration := v.End.Sub(v.Start)
-		at.Duration = float64(duration)
+		// Convert from nanosecond to millisecond
+		at.Duration = float64(duration) / float64(time.Second)
```
Comment (ybrill) [inline rest/model/test.go:105]: I think the time library has a built in conversion, something like
```
at.Duration = float64(duration.Milliseconds())
```

Comment (khelif96) [reply rest/model/test.go]: Thanks

File: rest/model/test.go [hunk 2]
```diff
@@ -101,7 +101,7 @@ func (at *APITest) BuildFromService(st interface{}) error {
 		at.StartTime = utility.ToTimePtr(v.Start)
 		at.EndTime = utility.ToTimePtr(v.End)
 		duration := v.End.Sub(v.Start)
-		at.Duration = float64(duration)
+		at.Duration = float64(duration.Milliseconds())
```
Comment (ybrill) [inline rest/model/test.go:104]: are you sure we're looking for milliseconds, though? In the previous case it looked to me like `duration` is the difference between two "python times" which are [in seconds](https://github.com/evergreen-ci/utility/blob/9e6c58062307caf4db647fba249bbe042b633ed8/time.go#L48).

Comment (khelif96) [reply rest/model/test.go]: Oof nice catch.

**Review by ybrill — APPROVED** [review]

**Pull request merged**